### PR TITLE
1712

### PR DESCRIPTION
**Pull request by jackgrossman18**
ref #1712

### CODE REVIEW

File: modules/Hoot/ui/sidebar/advancedOpts.js [hunk 1]
```diff
@@ -291,6 +294,49 @@ export default class AdvancedOpts {
...
+                            this.showing = false;
+                        }
+                        else {
+                            if (d.input === 'checkbox') {
+                                getDescId.classed('hoot-form-field-checkbox', false);
+                                getDescId.classed('hoot-form-field-checkbox-clicked', true );
+
+                            }
+                                getDescId
+                                    .append('div')
```
Comment (maxgrossman) [inline modules/Hoot/ui/sidebar/advancedOpts.js]: image

There seems to be a state in which having 2 dropdowns open leads to adding divs over and over if you click on one of the info buttons.

I think we can avoid this if we don't append on click, but show a div that we add in the same way we do this button  (d3 data binding cycle).

More succinctly I think you...

1. add the description div, not in the callback, but right underneath where 339 following d3 data binding flow
2. add a class for hidden to the div (there should be one we have already in the ui), if not, a display = none
3. then in the callback, all you do is set the value of the class/display property that shows/hides the div as the inverse of what it is `selection.classed('name-of-class, !selection.classed('name-of-class')`

File: css/80_app.css [hunk 2]
```diff
@@ -2198,9 +2198,55 @@ div.combobox {
...
+.adv-top {
+    margin-top: 10px;
+}
+
+.adv-opt-icon {
+    opacity: .5;
+}
+
+.hoot-field-label-button {
+        border: 1px solid #ccc;
```
Comment (maxgrossman) [inline css/80_app.css]: I think it looks fine without the border. adding the border creates the appearance of a double :rainbow:, oops i mean border!

**Review by maxgrossman — CHANGES_REQUESTED** [review]
See the comments above.

**Comment by jackgrossman18** [comment]
@maxgrossman requested changes made

**Comment by maxgrossman** [comment]
This is some serious nitpick, sorry, but I also noticed that "active" css is being applied at times when we might not expect it to. 

I think maybe we could only apply a tint to the background color "on hover"
hover_css
.

**Comment by jackgrossman18** [comment]
@maxgrossman

**Comment by jackgrossman18** [comment]
@maxgrossman ready to merge :triumph:

**Review by maxgrossman — APPROVED** [review]
:cake:

**Pull request merged**